packages/gp2-components/src/organisms/DashboardUserCard.tsx

```diff
@@ -0,0 +1,86 @@
+import { gp2 } from '@asap-hub/model';
+import { network } from '@asap-hub/routing';
+import { css } from '@emotion/react';
+
+import {
+  Card,
+  Avatar,
+  Ellipsis,
+  Paragraph,
+  ImageLink,
+  LinkHeadline,
+  TagList,
+  pixels,
+} from '@asap-hub/react-components';
+
+const { perRem } = pixels;
+const NUMBER_OF_TAGS_TO_DISPLAY = 5;
+
+const cardStyles = css({
+  padding: `${32 / perRem}em ${24 / perRem}em`,
+});
+
+const containerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  gap: `${15 / perRem}em`,
+});
+
+const avatarStyles = css({
+  width: `${96 / perRem}em`,
+  height: `${96 / perRem}em`,
+});
+
+const roleStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+});
+
+type DashboardRecommendedUsersProps = {
+  user: gp2.UserResponse;
+};
+
+const DashboardRecommendedUsers: React.FC<DashboardRecommendedUsersProps> = ({
+  user,
+}) => (
+  <Card key={user.id} overrideStyles={cardStyles}>
+    <div css={containerStyles}>
+      <ImageLink link={network({}).users({}).user({ userId: user.id }).$}>
```

Comment:
importing `import { gp2 as gp2Routing } from '@asap-hub/routing';` at the top

```suggestion
      <ImageLink link={gp2Routing.users({}).user({ userId: user.id }).$}>
```